test/unit/collector/remote-method.test.js

```diff
@@ -351,10 +351,10 @@ tap.test('when posting to collector', (t) => {
       })
     })
 
-    t.test('should default to deflated compression', (t) => {
+    t.test('should default to gzip compression', (t) => {
```

Comment:
do we need a test that tests when we set it to deflate that it still works?

Reply:
Test added now.